scripts/m4t/predict/README.md

```diff
@@ -39,6 +39,15 @@ python scripts/m4t/predict/predict.py <input_text> t2st <tgt_lang> --src_lang <s
 python scripts/m4t/predict/predict.py <path_to_input_audio> asr <tgt_lang>
 ```
 
+Note that it takes 16kHz audio now. Here's how you could resample your audio:
+```python
+import torchaudio
+resample_rate = 16000
+waveform, sample_rate = torchaudio.load('input_audio.wav')
```

Comment:
nit: To remain consistent with the README, change ‘input_audio.wav’ to <path_to_input_audio>